### PR TITLE
Update minion restart section in FAQ doc for windows

### LINKED ISSUE #41116: FAQ has wrong instructions for upgrading Windows minion.
### Description of Issue/Question
FAQ has wrong instructions for upgrading Windows minion.

In the FAQ it says as follows

> However, it requires more advanced tricks to upgrade from legacy version of Salt (before 2016.3.0), where executing commands in the background is not supported:

```
Restart Salt Minion:
  cmd.run:
{%- if grains['kernel'] == 'Windows' %}
    - name: 'C:\salt\salt-call.bat --local service.restart salt-minion'
{%- else %}
    - name: 'salt-call --local service.restart salt-minion'
{%- endif %}
    - bg: True
    - onchanges:
      - pkg: Upgrade Salt Minion
```
My present understanding is that a windows minion will not require a restart because when the upgrade occurs the old minion will get killed by the installer. Therefore the FAQ should indicated that Windows minions should not be restarted and the suggested Jinja code should look something like:

```
{%- if grains['kernel'] != 'Windows' %}
Restart Salt Minion:
  cmd.run:
    - name: 'salt-call --local service.restart salt-minion'
    - bg: True
    - onchanges:
      - pkg: Upgrade Salt Minion
{%- endif %}
```

### PR DESCRIPTION
**Pull request by rallytime**
Fixes #41116

@twangboy Please look this over and confirm that I have this correct.

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -319,7 +319,27 @@ Restart using states
...
+    {%- if grains['os'] != 'Windows' %
+    Restart Salt Minion:
+      cmd.run:
+        - name: 'salt-call --local service.restart salt-minion'
+        - bg: True
+        - onchanges:
+          - pkg: Upgrade Salt Minion
+    {%- endif %}
+
+Note that restarting the salt-minion service on Windows operating systems is
```
Comment (vutny) [inline doc/faq.rst]: Maybe format ``salt-minion`` as monospaced?

**Review by vutny — COMMENTED** [review]
Good finding!

Also, I think this should resolve issue #5721.

File: doc/faq.rst (hunk 1 above)
Comment (rallytime) [inline doc/faq.rst]: Yeah for sure. I've updated that. Thanks!

**Comment by cachedout** [comment]
@twangboy Were you able to look at this?

**Review by twangboy — APPROVED** [review]

**Pull request merged**